pkg/registration/hub/importer/renderers.go

```diff
@@ -0,0 +1,93 @@
+package importer
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/ghodss/yaml"
+	authv1 "k8s.io/api/authentication/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	clientcmdapiv1 "k8s.io/client-go/tools/clientcmd/api/v1"
+	"k8s.io/utils/ptr"
+
+	sdkhelpers "open-cluster-management.io/sdk-go/pkg/helpers"
+
+	"open-cluster-management.io/ocm/pkg/operator/helpers/chart"
+)
+
+func RenderBootstrapHubKubeConfig(
```

Comment:
could we consider to move the rednerers.go to the open-cluster-management.io/sdk-go? 
the other customized component like import-controller could consumer the same render func too.

Reply:
I am still thinking whether in import-controller, we should import ocm so we could reuse not only render but importer as well.